docs/examples/mtls/README.md

```diff
@@ -0,0 +1,43 @@
+# mTLS Example
+
+You can enable mTLS by providing the necessary TLS certificates and keys in Secret objects.
+You must set `.spec.tls.secretName` to the name of a secret containing the RabbitMQ server's TLS certificate and key,
+and set `spec.tls.caSecretName` to the name of a secret containing the certificate of the Certificate Authority which
+has signed the certificates of your RabbitMQ clients.
+
+First, you need to create the server Secret like this (assuming you already have `server.pem` and `server-key.pem` files):
+
+```shell
+kubectl create secret tls tls-secret --cert=server.pem --key=server-key.pem
+```
+
+Next, you must create the Secret containing the CA's certificate like this (assuming you already have a `ca.pem` file):
+
+```shell
+kubectl create secret generic ca-secret --from-file=ca.crt=ca.pem
+```
+
+These Secrets can also be created by [Cert Manager](https://cert-manager.io/).
+
+Once the secrets exist, you can deploy this example like this:
```

Comment:
What is `this example` that we are talking about? Do we link to the the example yaml?

Reply:
If the user's reading this text, they're already in `this example` at `docs/examples/mtls`. Is there anything you think I should add?

Reply:
Ah I see. I wasn't aware that this will be on an example page. Maybe just rephrase this to `you can deploy this example as follows:` to avoid repetition of this. Ignore if you think it's ok.